apps/voting/app/src/App.js

```diff
@@ -1,276 +1,115 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react'
-import { AragonApi, useAragonApi } from '@aragon/api-react'
+import React from 'react'
 import { Main, SidePanel } from '@aragon/ui'
 
-import { VOTE_YEA } from './vote-types'
-import { EMPTY_CALLSCRIPT } from './evmscript-utils'
-import tokenAbi from './abi/token-balanceOfAt.json'
-
 import EmptyState from './screens/EmptyState'
 import Votes from './screens/Votes'
 import VotePanelContent from './components/VotePanelContent'
 import NewVotePanelContent from './components/NewVotePanelContent'
 import AutoLink from './components/AutoLink'
 import AppLayout from './components/AppLayout'
 import NewVoteIcon from './components/NewVoteIcon'
+import LocalIdentityBadge from './components/LocalIdentityBadge/LocalIdentityBadge'
 
-import { NetworkContext, SettingsContext } from './app-contexts'
-import { IdentityProvider } from './components/IdentityManager/IdentityManager'
-import { isVoteOpen, voteTypeFromContractEnum } from './vote-utils'
-import { shortenAddress, transformAddresses } from './web3-utils'
-import { useNow } from './utils-hooks'
-import appStateReducer from './app-state-reducer'
+import { IdentityProvider } from './identity-manager'
+import { SettingsProvider } from './vote-settings-manager'
+import { transformAddresses } from './web3-utils'
+import { VotingAppProvider, useVotingApp } from './voting-app'
```

Comment:
Before I renamed it `VotingAppProvider` / `useVotingApp` / `voting-app.js`, I was using something really similar to what you suggest: `AppLogicProvider` / `useAppLogic` / `app-logic.js`. I changed it because I feared it was too generic, but the idea was to avoid adding another “appState”, and to have a name as generic as `App.js` so we can use the same in every app.

Let me know if you think that “voting” should stay in the name, otherwise I think we can use these names. :+1:

Reply:
I do like the simplicity and obviousness of `AppLogicProvider` / `useAppLogic` / `app-logic.js`.

My main goal is that it should be immediately obvious to someone new what a particular file is used for. `voting-app.js` is quite ambiguous, and also somewhat redundant.

Reply:
Agreed, done :+1: